dev-info/README.md

```diff
@@ -13,23 +13,12 @@ GitHub archive:
 
 PyPI:
 
-* make sure you have build and twine installed:
-    * `python3 -m pip install --upgrade build`
-    * `python3 -m pip install --upgrade twine`
-* OR use a conda env
-    * `conda env create -f dev-info/virocon-pip-build.yml` (create build env, if not done yet)
-    * `conda env update -f dev-info/virocon-pip-build.yml --prune` (update the build env if necessary)
-    * `conda activate virocon-pip-build`
-* `python3 -m build`
-* `python3 -m twine upload --repository testpypi dist/*`
-* `python3 -m twine upload dist/*`
-* in case of errors/questions consult https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives
-
-PyPI(old approach, uploads only a wheel):
-
-* `python setup.py bdist_wheel`
-* `twine upload dist/* -r pypitest`
-* `twine upload dist/*`
+* testpypi
+  * `uv publish --index testpypi`
+* pypi
+  * `uv publish`
+* The commands with ask for username and password.
+  Use `__token__` as username and your access token as password
```

Comment:
Can you explain this to me and possibly make it clearer in the readme too? What's __token__ and how do I get it?

Reply:
https://pypi.org/help/#apitoken